reana_commons/config.py

```diff
@@ -67,12 +67,13 @@
 }
 """REANA Workflow Controller address."""
 
-K8S_MAXIMUM_CONCURRENT_JOBS = 10
-"""Upper limit on concurrent jobs running in the cluster."""
+REANA_MAX_CONCURRENT_BATCH_WORKFLOWS = int(os.getenv(
```

Comment:
Configured from cluster creation time, see Helm `values.yaml` https://github.com/reanahub/reana/pull/261/commits/2649d55063f2e255be1a9c7656fe50612fb2e613.

Configurable at runtime by changing the environment variable.